multi_agent_path_planning/lifelong_MAPF/task_allocator.py

```diff
@@ -43,7 +43,15 @@ def allocate_tasks(self, tasks: TaskSet, agents: AgentSet) -> AgentSet:
 
         # Assign each agent a task
         for agent, task in zip(sampled_agents, sampled_tasks):
-            agent.set_task(task)
+            # TODO: make this more elegent, we dont want to assign tasks where the agent is on top of the start, unless we rework some of the initilization stuff, it creates issues with the planner which assumes there is a path required
+            if (
+                agent.get_loc()[0] != task.start[0]
+                and agent.get_loc()[1] != task.start[1]
```

Comment:
I don't think this is the same as checking if two tuples are not equal.

Reply:
I'm confused on why this is needed. I think we actually really want to give an agent a task if it's at the start. We should just maybe set it's goal to the task end, rather than the start, at some point in the process.